content/response-to-reviewers.md

```diff
@@ -110,8 +115,16 @@ See [GH142](https://github.com/greenelab/meta-review/issues/142) and [GH170](htt
 > `We initiated the Deep Review` → when?
 Would be nice to give some date at this point.
 
+Deep Review as started in August 2016.
```

Comment:
Is this meant to be a quote?

Reply:
It wasn't meant to be.  I fixed a typo and rephrased.